scipy/stats/_continuous_distns.py

```diff
@@ -3079,6 +3079,90 @@ def _stats(self, mu):
 invgauss = invgauss_gen(a=0.0, name='invgauss')
 
 
+class norminvgauss_gen(rv_continuous):
+    r"""A Normal Inverse Gaussian continuous random variable.
+
+    %(before_notes)s
+
+    Notes
+    -----
+    The probability density function for `norminvgauss` is:
+
+    .. math::
+
+        f(x; a, b) = 
+        \frac{a  \exp\left(\sqrt{a^2 - b^2} + b x\right)}
+        {\pi \sqrt{1 + x^2}} \, K_1\left(a * sqrt{1 + x^2}\right)
+
+    where `x` is a real number, the parameter `a` is the tail heaviness
+    and `b` is the asymmetry parameter satisfying `a > 0` and `abs(b) <= a`.
+    `K_1` is the modified Bessel function of second kind (`scipy.special.k1`).
+
+    Due to the scaling properties of the distributions, note that
+
+    .. math::
+
+        norminvgauss.pdf(x, a d, b d, loc=\mu, scale=d) =
+        \frac{a  d \exp\left(d \sqrt{a^2 - b^2} + b (x - \mu)\right)}
+        {\pi \sqrt{d^2 + (x - \mu)^2}} \,
+        K_1\left(a sqrt{d^2 + (x - \mu)^2}\right),
+
+    which is another frequently used parametrisation of the distribution using
+    an additional scale and location parameter.
```

Comment:
I would say it is a bit more complicate here since the scale parameter affects the other parameters of the distribution in a certain way, but let's drop it. It is not too hard to figure out

Reply:
Of there's a common parametrization, which is different from the generic loc-scale 
(eg scaling of shape parameters etc), ba Rice example is IMO a good one: 
docs.scipy.org /doc/scipy/reference/generated/scipy.stats.rice.html